lxc/list.go

```diff
@@ -781,3 +828,43 @@ func (c *cmdList) NumberOfProcessesColumnData(cInfo api.InstanceFull) string {
 func (c *cmdList) locationColumnData(cInfo api.InstanceFull) string {
 	return cInfo.Location
 }
+
+func matchByType(cInfo *api.Instance, query string) bool {
```

Comment:
Same low-priority request as below, but for `con and vm` in addition to `container and virtual-machine`.

Reply:
@toby63 Thanks for taking the time to look through the pullrequest and write down your thoughts on it. Definitely helped me get my bearings. : )

Reply:
Thank you, but I almost did nothing useful :sweat_smile: 
Instead I have to thank you for providing this and being so patient about all the things we comment :wink:.